overlays/server-os-debian/etc/initramfs-tools/scripts/init-premount/stiefellink

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+PREREQ="udev"
+prereqs()
+{
+	echo "$PREREQ"
+}
+
+case $1 in
+prereqs)
+	prereqs
+	exit 0
+	;;
+esac
+
+case "$(cat /proc/cmdline)" in
+  *stiefel_link*)
+	echo "wir werden gestiefelt"
+
+	for x in $(cat /proc/cmdline); do
+		case "$x" in
+			stiefel_link=*)
+				stiefel_link="${x#stiefel_link=}"
+				;;
+			stiefel_nbdhost=*)
+				stiefel_link="${x#stiefel_nbdhost=}"
+				;;
+			stiefel_nbdname=*)
+				stiefel_link="${x#stiefel_nbdname=}"
+				;;
```

Comment:
in all 3 cases the evaluation result is assigned to `stiefel_link`... why does this even work?